express-post-json/index.js

```diff
@@ -0,0 +1,30 @@
+const express = require('express');
+const app = express();
+
+let nextID = 1;
+const grades = {};
+
+app.get('/api/grades', (req, res) => {
+  const gradesArray = [];
+  for (const key in grades) {
+    gradesArray.push(grades[key]);
+  }
+  if (!gradesArray.length === 0) {
+    res.status(404).send('No grades :(');
```

Comment:
An empty list is not a `404` scenario. Also, I don't think this conditional expression will ever evaluate to `true`. In any case, you can remove this condition.